WooCommerce/Classes/ViewRelated/Orders/Order Filters/FilterOrderListViewModel.swift

```diff
@@ -47,9 +47,10 @@ final class FilterOrderListViewModel: FilterListViewModel {
 
     /// - Parameters:
     ///   - filters: the filters to be applied initially.
-    init(filters: Filters) {
-        orderStatusFilterViewModel = OrderListFilter.orderStatus.createViewModel(filters: filters)
-        dateRangeFilterViewModel = OrderListFilter.dateRange.createViewModel(filters: filters)
+    ///   - allowedStatuses: the statuses that will be shown in the filter list.
+    init(filters: Filters, allowedStatuses: [OrderStatus]) {
+        orderStatusFilterViewModel = OrderListFilter.orderStatus.createViewModel(filters: filters, allowedStatuses: allowedStatuses)
+        dateRangeFilterViewModel = OrderListFilter.dateRange.createViewModel(filters: filters, allowedStatuses: allowedStatuses)
```

Comment:
nit: it's a bit odd seeing `allowedStatuses` passed to create a date range view model 🤔 I wonder if moving the `allowedStatuses` to `Filters` or somewhere else might help

Reply:
I can make it optional eventually, WDYT?

> I wonder if moving the allowedStatuses to Filters or somewhere else might help

Do you mean under the `Filters` of `FilterOrderListViewModel`? I don't see a great benefit 🤔

Reply:
I couldn't think of a good way to remove the `allowedStatuses` from view model creation given the enum usage, no worries about it! 😄